txcache/virtualAccountBalance.go

```diff
@@ -13,11 +13,14 @@ type virtualAccountBalance struct {
 // virtualAccountBalance is used in two scenarios:
 // when validating a proposed block (on the OnProposedBlock notification), where we create a virtualBalance for each account;
 // inside a virtual record
-func newVirtualAccountBalance(initialBalance *big.Int) *virtualAccountBalance {
+func newVirtualAccountBalance(initialBalance *big.Int) (*virtualAccountBalance, error) {
+	if initialBalance == nil {
+		return nil, errNilBalance
```

Comment:
:crossed_fingers: hopefully never reached. I know it's not reached in long tests, but we also have to see in system tests.